spawn_worker/build.rs

```diff
@@ -11,13 +13,13 @@ fn main() {
         .warnings_into_errors(true)
         .emit_rerun_if_env_changed(true);
 
-    if !cfg!(target_os = "windows") {
+    if !env::var("TARGET").unwrap().contains("windows") {
```

Comment:
Why didn't `target_os` work anymore?

Reply:
cfg is evaluated during compilation. When cross compiling - you're compiling on linux